### PR TITLE
Added scripts for the teachers we missed yesterday in the 2016-11-10-…

### PR DESCRIPTION
**Pull request by aoby**
…moving-forward-together email

### CODE REVIEW

File: bin/mail/2016-11-10-moving-forward-together-remaining-teachers/generate-teachers [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      "united staes of america" OR
+      unitedstatesofamerica OR
+      "unitied states" OR
+      "united sates of america" OR
+      "united stares" OR
+      "untied states of america" OR
+      "united states of amrica" OR
+      "united stats of amarica" OR
+      "unided states of america" OR
+      "United Atates"
```
Comment (ashercodeorg) [inline bin/mail/2016-11-10-moving-forward-together-remaining-teachers/generate-teachers:52]: Impressive that we are catching this.

**Comment by ashercodeorg** [comment]
It would be nice to see our test email addresses in a test to confirm we aren't resending to the recipients from yesterday.

**Comment by ashercodeorg** [comment]
LGTM, pending test.

**Comment by aoby** [comment]
I just verified manually that our test emails are in the list, in `pegasus-console`:

```
irb(main):002:0> previous_emails = Set.new
=> #<Set: {}>
irb(main):003:0> TEMPLATE_NAME = '2016-11-10-moving-forward-together'.freeze
=> "2016-11-10-moving-forward-together"
irb(main):004:0> message_id = DB[:poste_messages].where(name: TEMPLATE_NAME).first[:id]
=> 259
irb(main):005:0> DB[:poste_deliveries].where(message_id: message_id).each do |row|
irb(main):006:1*   previous_emails.add(row[:contact_email])
irb(main):007:1> end
=> #<Sequel::Mysql2::Dataset: "SELECT * FROM `poste_deliveries` WHERE (`message_id` = 259)">
irb(main):008:0> previous_emails.include? 'andrew+us@code.org'
=> true
irb(main):009:0> previous_emails.include? 'asher+us@code.org'
=> true
```

**Comment by ashercodeorg** [comment]
LGTM. Thanks!

**Pull request merged**